packages/manager/src/features/Account/Quotas/Quotas.tsx

```diff
@@ -41,85 +42,95 @@ export const Quotas = () => {
+                    to: flags?.iamRbacPrimaryNavChanges
+                      ? '/quotas'
+                      : '/account/quotas',
+                  });
+                }}
+                options={
+                  sortedS3Endpoints?.map((location) => ({
+                    label: location.label,
+                    value: location.value,
+                  })) ?? []
+                }
+                placeholder={
+                  isFetchingLocations
+                    ? `Loading S3 endpoints...`
+                    : 'Select an Object Storage S3 endpoint'
+                }
+                searchable
+                sx={{ flexGrow: 1, mr: 2 }}
+              />
+            </Stack>
+            <Divider spacingBottom={40} spacingTop={40} />
+            <Stack
+              direction="row"
+              justifyContent="space-between"
+              marginBottom={2}
+            >
+              <Typography variant="h3">Quotas</Typography>
+            </Stack>
+            <Typography>
+              If you need to increase a quota, select{' '}
+              <strong>Request Increase</strong> from the Actions menu. Usage can
+              also be found using third-party tools like{' '}
+              <Link to="https://techdocs.akamai.com/cloud-computing/docs/using-s3cmd-with-object-storage#check-disk-usage-by-bucket">
+                s3cmd
+              </Link>
+              .
+            </Typography>
+            <Stack direction="column" spacing={2}>
+              <QuotasTable
+                selectedLocation={selectedLocation}
+                selectedService={{
+                  label: 'Object Storage',
+                  value: 'object-storage',
+                }}
+              />
+            </Stack>
           </Stack>
-          <Divider spacingBottom={40} spacingTop={40} />
-          <Stack
-            direction="row"
-            justifyContent="space-between"
-            marginBottom={2}
-          >
-            <Typography variant="h3">Quotas</Typography>
-          </Stack>
-          <Typography>
-            If you need to increase a quota, select{' '}
-            <strong>Request Increase</strong> from the Actions menu. Usage can
-            also be found using third-party tools like{' '}
-            <Link to="https://techdocs.akamai.com/cloud-computing/docs/using-s3cmd-with-object-storage#check-disk-usage-by-bucket">
-              s3cmd
-            </Link>
-            .
-          </Typography>
-          <Stack direction="column" spacing={2}>
-            <QuotasTable
-              selectedLocation={selectedLocation}
-              selectedService={{
-                label: 'Object Storage',
-                value: 'object-storage',
-              }}
-            />
-          </Stack>
-        </Stack>
-      </Paper>
+        </Paper>
+      ) : (
+        <Notice
+          text={getRestrictedResourceText({
+            action: 'view',
+            resourceType: 'Quotas',
```

Comment:
Since this is the landing page for all quotas and we're showing this if they don't have access to the overall feature, should this banner be plural ("Quotas" instead of "this Quota")?

```suggestion
            isSingular: false,
            resourceType: 'Quotas',
```

![Screenshot 2025-08-19 at 1 26 47 PM](https://github.com/user-attachments/assets/5996991d-d50b-4e74-84fc-6d52a7d02cd1)